### DISCUSSION TITLE
[10일차 -윤경] button의 속성

### DISCUSSION THREAD

**Discussion by skyhanull** [zzangCS/zzangInterview#34] [프론트 면접 질문]
button의 속성에는 무엇이 있을까요?

**Comment by skyhanull** [comment]
버튼의 속성에는 
디폴트로 submit이 되어있고 그외에 reset,button속성이 있습니다

**Comment by ez1n** [comment]
button에는 submit, reset, button 세 가지 type을 지정할 수 있는 일반적으로 submit이 기본값이기 때문에 type을 명시해주지 않으면 submit처리가 일어나게 된다.
